.circleci/config.yml

```diff
@@ -180,6 +180,51 @@ jobs:
           gsutil rsync -d -r dataproc_bootstrap gs://moz-fx-data-prod-airflow-dataproc-artifacts/bootstrap
           gsutil rsync -d -r jobs gs://moz-fx-data-prod-airflow-dataproc-artifacts/jobs
 
+  sync-dags-repo:
+    executor: docker/machine
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - "e4:30:50:41:53:f0:d6:3a:bb:c9:38:54:2d:ca:56:41"
+      - run:
+          name: 🤖 Update DAGs repository
+          command: |
+            ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
```

Comment:
Is this needed here as well? https://github.com/mozilla/bigquery-etl/blob/f0c53d7afebaf1e6d8c10754b7dcdffdab0a9a37/.circleci/config.yml#L429-L438

Reply:
Not required because we clone telemetry-airflow which is public. The SSH config override was necessary in BQetl because we clone private-bigquery-etl.